Zend/zend_alloc.c

```diff
@@ -663,35 +696,47 @@ static zend_always_inline int zend_mm_bitset_is_free_range(zend_mm_bitset *bitse
 
 static void *zend_mm_chunk_alloc_int(size_t size, size_t alignment)
 {
-	void *ptr = zend_mm_mmap(size);
-
+#ifdef _WIN32
+	void *ptr = NULL;
+	size_t offset;
+#ifndef _WIN64
```

Comment:
I'm confused by this. I think the `_WIN32` and `_WIN64` macros are mutually exclusive, and you don't need to add code for `_WIN64`. (WIN32 without an underscore is the macro for the existence of windows 32 apis)

- _WIN64 has support for mmap (not targeting legacy platforms?), and we use mmap/munmap on 64-bit windows

https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

> `_WIN32` Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, or x64. Otherwise, undefined.
>
> `_WIN64` Defined as 1 when the compilation target is 64-bit ARM or x64. Otherwise, undefined.

Reply:
I used `_WIN32` as before for detecting any Windows (x86/x64), anything to adjust?

Reply:
> `_WIN32` Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, or x64. Otherwise, undefined.

Oh. I misread that and missed the part about 64-bit and x64.